controllers/tokenController.js

```diff
@@ -0,0 +1,40 @@
+const User = require("../models/users");
+const sequelize = require("../database/db");
+let crypto = require("crypto");
+const Access_Token = require("../models/access_token");
+
+module.exports = {
+  createToken: async (req, res) => {
+    try {
+      const { username } = req.body;
+      const found = await User.findOne({
+        where: {
+          username,
+        },
+      });
+      if (!found) {
+        return res.status(400).json({ message: "Invalid credentials" });
+      }
+
+      const user_id = found.id;
+
+      const foundedUsername = found.username;
+      let token = crypto
+        .createHash("md5")
+        .update(foundedUsername)
+        .digest("hex");
+
+      const expiryTime = new Date();
+      const exTime = expiryTime.setHours(expiryTime.getHours() + 1);
```

Comment:
define hours value in env file and use from there

Reply:
has been done.